webrtc-charter.html

```diff
@@ -459,7 +459,7 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 	<dl>
         <dt><a href="https://www.w3.org/media-wg/">Media Working Group</a></dt>
         <dd>The Media Working Group develops client-side media processing APIs and features that are likely to intersect with the WebRTC Working Group developments.</dd>
-        <dt class="todo">WebTransport Working Group</dt>
+        <dt><a href="https://www.w3.org/groups/wg/webtransport">WebTransport Working Group</dt>
         <dd>The WebTransport Working Group develops a client/server API that has been inspired by WebRTC datachannels and may inform their future evolution.</dd>
```

Comment:
```suggestion
        <dd>The WebTransport Working Group develops a client/server API that provides networking capabilities similar to how WebRTC datachannels have been used.</dd>
```
Does this work better @aboba ? if not, could you suggest an alternative wording? or are you suggesting we should not list the  WebTransport WG at all?